test/middleware_test.rb

```diff
@@ -29,11 +29,12 @@ def new_processor(boss)
 
 class SidekiqPost63
   def new_processor(boss)
-    config = Sidekiq
-    config[:queues] = ['default']
-    config[:fetch] = Sidekiq::BasicFetch.new(config)
-    config[:error_handlers] << Sidekiq.method(:default_error_handler)
-    ::Sidekiq::Processor.new(config) { |processor, reason = nil| }
+    configSidekiq = Sidekiq.configure_server do |config|
```

Comment:
Let's name this `sidekiq_config` or something similar to stick with snake casing